Mission 3/NMCA_003/NMCA_003_strings.lua

```diff
@@ -1,9 +1,455 @@
+M1Enginners2 = {
+    {text = '[Benson]: The transports are inbound.', vid = 'N03_M1Enginners2.sfd', bank = 'N03_VO', cue = 'M1Enginners2', faction = 'UEF'},
+}
+
+-- Engineers start repairing the ship
+M1Enginners3 = {
+    {text = '[Benson]: Scanning the ship for damage.', vid = 'N03_M1Enginners3.sfd', bank = 'N03_VO', cue = 'M1Enginners3', faction = 'UEF'},
+}
+
+-- Few seconds after, assign objective to reclaim crystal
+M1CrystalsObjective = {
+    {text = '[Benson]: Ship is very damaged, to repair it we must reclaim some crystals, etc etc...', vid = 'N03_M1CrystalsObjective.sfd', bank = 'N03_VO', cue = 'M1CrystalsObjective', faction = 'UEF'},
+}
+
+-- Remind player to reclaim the crystals
+M1CrystalsReminder = {
+    {text = '[Benson]: Commander, we really need those crystals, reclaim them ASAP.', vid = 'N03_M1CrystalsReminder.sfd', bank = 'N03_VO', cue = 'M1CrystalsReminder', faction = 'UEF'},
+}
+
+-- When the first crystal is reclaimed
+M1FirstCrystalReclaimed = {
+    {text = '[Nichols]: First reclaim reclaimed, get some more.', vid = 'N03_M1FirstCrystalReclaimed.sfd', bank = 'N03_VO', cue = 'M1FirstCrystalReclaimed', faction = 'UEF'},
+}
+
+-- Both crystals reclaimed, objective completed (map didnt expand yet)
+M1CrystalsReclaimed1 = {
+    {text = '[Nichols]: Both crystals reclaimed, we will need more.', vid = 'N03_M1CrystalsReclaimed1.sfd', bank = 'N03_VO', cue = 'M1CrystalsReclaimed1', faction = 'UEF'},
+}
+
+-- Both crystals reclaimed, objective completed (map has already expanded)
+M1CrystalsReclaimed2 = {
+    {text = '[Nichols]: Both crystals reclaimed.', vid = 'N03_M1CrystalsReclaimed2.sfd', bank = 'N03_VO', cue = 'M1CrystalsReclaimed2', faction = 'UEF'},
+}
+
+
+
+-- Warn player about incoming naval attack
+M1AirAttack = {
+    {text = '[Benson]: Commander, we\'re picking up air units incoming from the west.', vid = 'N03_M1AirAttack.sfd', bank = 'N03_VO', cue = 'M1AirAttack', faction = 'UEF'},
+}
+
+-- Warn player about incoming naval attack
+M1NavalAttack = {
+    {text = '[Benson]: Commander, we\'re picking up a group of naval units approaching from the west.', vid = 'N03_M1NavalAttack.sfd', bank = 'N03_VO', cue = 'M1NavalAttack', faction = 'UEF'},
+}
+
+-- Map expands by the timer, this dialogue is followed by cinematics intro of M2
+M1MapExpansion = {
+    {text = '[Nichols]: The crystals won\'t be enough to repair the ship. We will need more. Scanning the surrounding area for them...', vid = 'N03_M1MapExpansion.sfd', bank = 'N03_VO', cue = 'M1MapExpansion', faction = 'UEF', delay = 5},
+}
+
+
+
+-- Primary Objective
+M1P1Title = 'Protect the crashed ship'
+M1P1Description = 'Ensure the crashed ship survives.'
+
+-- Primary Objective
+M1P2Title = 'Destroy the southern Aeon base'
+M1P2Description = 'The Aeon units are sending attacks from this position. Destroy the base.'
+
+-- Primary Objective
+M1P3Title = 'Reclaim these things'
+M1P3Description = 'Reclaim these things so the ship can get better.'
```

Comment:
Reclaim the resource rich crystals to obtain the Dilithium and Neutronium need to get the ships systems online again.